crates/solidity/testing/solc/src/utils/json_api.rs

```diff
@@ -1,14 +1,20 @@
 use std::collections::HashMap;
```

Comment:
While it's obvious for me now, I struggled to understand at first that these are the types for the JSON API for the `solc`.

I know it's not exactly related to the PR but could we at least add a link to https://docs.soliditylang.org/en/latest/using-the-compiler.html#compiler-input-and-output-json-description in the module comment? :pray:

Reply:
Sure thing.